documentation/docs/assertions/json/overview.md

```diff
@@ -6,6 +6,8 @@ sidebar_label: Overview
 
 To use these matchers add `testImplementation("io.kotest:kotest-assertions-json:<version>")` to your build.
 
+There exist copies of all matchers that validate a `File` or a `Path` instead of a `String` for the JVM platform.
```

Comment:
```suggestion
There are overloads of all matchers that validate a `File` or a `Path` instead of a `String` for the JVM platform.
```

Reply:
they're not technically overloads though as they're effectively distinct extension methods, no?

because, at least as far as a kotlin dev is concerned, extension functions execute *on* some extension. they have no need to know the implementation detail that it is actually just a parameter to a static function